CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.11.8]
+### Added
+- Support `buttonAppearance` with `outline` in `ButtonDropdown`
```

Comment:
please include this in 0.11.7 because 0.11.7 is not released yet
and also please update version in package.json to 0.11.7. thanks

Reply:
yeah, I have considered merging to 0.11.7